packages/launchpad/src/action-components/share-site-modal/index.tsx

```diff
@@ -14,11 +20,84 @@ interface ShareSiteModalProps {
 	site: SiteDetails | null;
 }
 
+interface ShareLink {
+	href: string;
+	className: string;
+	title: string;
+	icon: 'mail' | 'tumblr' | 'bluesky' | 'linkedin' | 'telegram' | 'reddit' | 'whatsapp' | 'x';
+	label: string;
+}
+
+const getShareLinks = ( siteUrl: string, text: string ): ShareLink[] => {
+	const encodedSiteUrl = encodeURIComponent( siteUrl );
+	const encodedText = encodeURIComponent( text );
+	return [
+		{
+			href: `mailto:?subject=${ encodedText }&body=${ encodedSiteUrl }`,
+			className: 'share-site-modal__modal-share-link',
+			title: __( 'Share via email', 'launchpad' ),
+			icon: 'mail',
+			label: __( 'Email', 'launchpad' ),
+		},
+		{
+			href: `http://www.tumblr.com/share/link?url=${ encodedSiteUrl }`,
+			className: 'share-site-modal__modal-share-link',
+			title: __( 'Share on Tumblr', 'launchpad' ),
+			icon: 'tumblr',
+			label: __( 'Tumblr', 'launchpad' ),
+		},
+		{
+			href: `https://bsky.app/intent/compose?text=${ encodedSiteUrl }`,
+			className: 'share-site-modal__modal-share-link',
+			title: __( 'Share on Bluesky', 'launchpad' ),
+			icon: 'bluesky',
+			label: __( 'Bluesky', 'launchpad' ),
+		},
+		{
+			href: `https://www.linkedin.com/shareArticle?mini=true&url=${ encodedSiteUrl }&title=${ encodedText }`,
+			className: 'share-site-modal__modal-share-link',
+			title: __( 'Share on LinkedIn', 'launchpad' ),
+			icon: 'linkedin',
+			label: __( 'LinkedIn', 'launchpad' ),
+		},
+		{
+			href: `https://t.me/share/url?url=${ encodeURIComponent(
+				encodedSiteUrl
+			) }&text=${ encodedText }`,
+			className: 'share-site-modal__modal-share-link',
+			title: __( 'Share on Telegram', 'launchpad' ),
+			icon: 'telegram',
+			label: __( 'Telegram', 'launchpad' ),
+		},
```

Comment:
This appears to be double-encoding for me:

<img width="475" alt="image" src="https://github.com/user-attachments/assets/b4942eaf-0abd-4359-8e1b-2958190963a2" />

If I manually update the URL then it looks okay:

<img width="498" alt="image" src="https://github.com/user-attachments/assets/c869cd8d-63b7-4290-8877-7cabc4d74cc5" />

Do we need the additional `encodeURIComponent` call?

Reply:
No we don't! That was a last minute copy pasta. Thanks for catch 🙇🏻

Reply:
Fixed!

Reply:
That's testing well for me, now 👍